Makefile

```diff
@@ -328,10 +328,7 @@ ifeq ($(RELEASE), 1)
       # O3 (inclusive)
       OPTLEVEL = -O3
     endif
-  endif
-  CXXFLAGS += $(OPTLEVEL)
 
-  ifeq ($(LTO), 1)
```

Comment:
Oh, wait, you didn't mean just that one line.
That's intentional. Moved line 346 to end of branch. Seamed strange break it for an assignment, if it does not change in the rest.

Reply:
Then I cannot turn off `LTO`, right?

Reply:
Yes, you can, only thing changes that 325,344 is one branch.

Reply:
Sorry, didn't see the context beyond diff.

Reply:
Understandable, neither did I, when you first asked.